Aliases/v8@5.1

```diff
@@ -1 +0,0 @@
-../Formula/v8.rb
```

Comment:
You have to create a new `v8@6.5` alias to make brew audit happy:

```
  * Formula has other versions so create a versioned alias:
      cd /usr/local/Homebrew/Library/Taps/homebrew/homebrew-core/Aliases
      ln -s ../Formula/v8.rb v8@6.5
```

Reply:
fixed.